src/System.Reflection.Primitives/src/System.Reflection.Primitives.builds

```diff
@@ -9,6 +9,10 @@
     <Project Include="System.Reflection.Primitives.csproj">
       <TargetGroup>netcore50</TargetGroup>
     </Project>
+    <Project Include="System.Reflection.Primitives.csproj">
+      <OSGroup>Windows_NT</OSGroup>
```

Comment:
You can remove the OSGroup for this guys. I recently updated all the builds files to remove it unless there is a need for a specific build for it, otherwise it is just IL with no OS dependencies that can build on any OS.